odkx-src/build-scripts.rst

```diff
@@ -67,6 +67,14 @@ If you are on :program:`Windows` use :file:`gradlew.bat` instead.
 
   If you are building with Android Studio, you will need to select the correct build variant. This is important when you don't have androidlibrary or androidcommon in your :ref:`build-scripts-directory-structure`. These are discussed more in the :ref:`next section <build-scripts-flavors>`.
 
+.. warning::
+
+  The Android tools have a release signature key unique to the ODK-X applications, communication between the applications are secured by this key and can happen when the application keys match. This ensures that the information between the applications cannot be intercepted by another application. 
```

Comment:
Minor grammar details: break up sentence with a period after applications and before communication. "Communication between the applications is secured" (communication is singular) "and can happen only when the application keys match". Also feel like something missing between information and information between. "information moving between" maybe?

Reply:
It probably makes sense to have  link to Google's description of how to app sign
https://developer.android.com/studio/publish/app-signing